packages/blocks/src/components/blockhub.ts

```diff
@@ -0,0 +1,648 @@
+  TodoIcon,
+} from '../page-block/utils/icons.js';
+import { centeredToolTipStyle, toolTipStyle } from './tooltip.js';
+import { assertExists } from '@blocksuite/global/utils';
+
+type BlockHubItem = {
+  flavour: string;
+  type: string | null;
+  name: string;
+  description: string;
+  icon: unknown;
+  toolTip: string;
+};
+
+const BLOCKHUB_TEXT_ITEMS = [
+  {
+    flavour: 'affine:paragraph',
+    type: 'text',
+    name: 'Text',
+    description: 'Start typing with plain text.',
+    icon: TextIcon,
+    toolTip: 'Drag to insert Text block',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h1',
+    name: 'Heading 1',
+    description: 'Heading in the largest font.',
+    icon: H1Icon,
+    toolTip: 'Drag to insert Heading 1', // TODO confirm
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h2',
+    name: 'Heading 2',
+    description: 'Heading in second largest font.',
+    icon: H2Icon,
+    toolTip: 'Drag to insert Heading 2',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h3',
+    name: 'Heading 3',
+    description: 'Heading in third largest font.',
+    icon: H3Icon,
+    toolTip: 'Drag to insert Heading 3',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h4',
+    name: 'Heading 4',
+    description: 'Heading in forth largest font.',
+    icon: H4Icon,
+    toolTip: 'Drag to insert Heading 4',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h5',
+    name: 'Heading 5',
+    description: 'Heading in fifth largest font.',
+    icon: H5Icon,
+    toolTip: 'Drag to insert Heading 5',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h6',
+    name: 'Heading 6',
+    description: 'Heading in sixth largest font.',
+    icon: H6Icon,
+    toolTip: 'Drag to insert Heading 6',
+  },
+  {
+    flavour: 'affine:code',
+    type: 'code',
+    name: 'Code Block',
+    description: 'Capture a code snippet.',
+    icon: CodeIcon,
+    toolTip: 'Drag to insert Code Block',
+  },
+  {
+    flavour: 'affine:paragraph',
+    type: 'quote',
+    name: 'Quote',
+    description: 'Capture a quote.',
+    icon: QuoteIcon,
+    toolTip: 'Drag to insert Quote',
+  },
+  {
+    flavour: 'affine:divider',
+    type: null,
+    name: 'Divider',
+    description: 'A visually divide block.',
+    icon: DividerIcon,
+    toolTip: 'Drag to insert Divider',
+  },
+];
+
+const BLOCKHUB_LIST_ITEMS = [
+  {
+    flavour: 'affine:list',
+    type: 'bulleted',
+    name: 'Bulleted List',
+    description: 'a simple bulleted list.',
+    icon: BulletedListIcon,
+    toolTip: 'Drag to insert Bulleted List.',
+  },
+  {
+    flavour: 'affine:list',
+    type: 'numbered',
+    name: 'Numbered List',
+    description: 'A list with numbering.',
+    icon: NumberedIcon,
+    toolTip: 'Drag to insert Numbered List.',
+  },
+  {
+    flavour: 'affine:list',
+    type: 'todo',
+    name: 'To-do List',
+    description: 'Track tasks with a to-do list.',
+    icon: TodoIcon,
+    toolTip: 'Drag to insert To-do List.',
+  },
+];
```

Comment:
There is a very similar configuration at `packages/blocks/src/page-block/utils/const.ts`. Will it come in handy?

Reply:
If I extend the configuration at `packages/blocks/src/page-block/utils/const.ts` with two more filed: `description`, `toolTip`, will it break it's original usage? It seems safe to do so, and it will be handy with only one more divider manually extended needed

Reply:
I find the item order is slightly different, this will affect the order to display, and list items are categorized differently, how about just keep it

Reply:
Now the icons and consts in `packages/blocks` have been moved to `packages/globals/config`